<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WatershedDialogBase</class>
 <widget class="QDialog" name="WatershedDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>341</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Batch Processor</string>
  </property>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="txtFill">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>55</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Fill Sink : </string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbLayer">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>190</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>25</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Elevation : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>89</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Flat : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFlat">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>85</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>119</y>
      <width>130</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Flow Direction : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFD">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>116</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>150</y>
      <width>150</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Flow Accumulation : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFAC">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>145</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>180</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Slope : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSlope">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>175</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="txtCellValue">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>50</y>
       <width>120</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Threashold Value : </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>(FA Cell Number)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtStream">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>16</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>74</x>
       <y>20</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Stream : </string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
